Entities/Common/Decaying/DecayQuantity.as

```diff
@@ -6,46 +6,48 @@
 // discarded each step
 
 const uint16 FREQUENCY = 2000;
+const uint16 QUICK_FREQUENCY = 30;
 const uint16 QUARTER = FREQUENCY / 4;
 
 void onInit(CBlob@ this)
 {
-  ScriptData@ script = this.getCurrentScript();
-  script.runFlags |= Script::tick_not_attached;
-  script.runFlags |= Script::tick_not_ininventory;
-  script.tickFrequency = FREQUENCY;
+	ScriptData@ script = this.getCurrentScript();
+	script.runFlags |= Script::tick_not_attached;
+	script.runFlags |= Script::tick_not_ininventory;
+
+	this.getCurrentScript().tickFrequency = (getRules().hasTag('quick decay') ? QUICK_FREQUENCY : FREQUENCY);
```

Comment:
it would be better to have a function that does this instead of having all those duplicate ternary expressions, but i'll merge anyway for now